app/uk/gov/hmrc/hmrcemailrenderer/templates/TemplateLocator.scala

```diff
@@ -118,6 +118,11 @@ trait TemplateLocator {
   lazy val all: Seq[MessageTemplate] = templateGroups.values.flatten.toSeq
 
   def findTemplate(templateId: String): Option[MessageTemplate] =
-    all.find(_.templateId == templateId)
+    all.find(_.templateId == templateId) orElse  {
```

Comment:
now that you changed, it is looks like we could simplify it even further :)
guess we can just do in one find: all.find(id => id == templateId || id == templateId.stripSuffix(...)), what do you think :) ?